datajoint/relational_operand.py

```diff
@@ -266,7 +308,7 @@ def __sub__(self, restriction):
         """
         return self & Not(restriction)
 
-    def restrict(self, restriction):
+    def restrict(self, arg):
```

Comment:
I liked it better to call it `restriction` rather than arbitrary `arg` as the parameter name.